### ISSUE TITLE
some music glitching on linux

### ISSUE THREAD

**Issue by legacycoder254** [KRTirtho/spotube#887]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

some music glitching on linux

### Expected Behavior

Seamless streaming

### Steps to reproduce

some music glitching on linux

### Operating System

Linux

### Spotube version

V3.00

### Installation source

Flathub (Flatpak)

**Comment by hexlocation** [comment]
Using the flatpak version of spotube here; experiencing severe audio issues too. From the volume dropping severely for a split second to the entire song hanging.

**Comment by KRTirtho** [comment]
Can you try turning off the "Normalize Audio" option?

**Comment by hexlocation** [comment]
> Can you try turning off the "Normalize Audio" option?

Yep, that worked. What does the Normalize Audio option do? I don't hear any difference from spotify and spotube with the normalize audio option disabled.

**Comment by KRTirtho** [comment]
It normalizes the volume across all tracks. But the mpv module doesn't work good enough sometimes
